server/models/tables.js

```diff
@@ -34,17 +34,52 @@ const tableSchema = new Schema({
     type: [String],
     default: []
   },
-  playersOnTable: {
-    type: [String],
-    default: []
-  },
+   playersOnTable: [{
```

Comment:
you had here a list of strings for all the nicknames of the users that in the table, if you want to add stuff that's good but no need username, password, email, isApprovedByEmail, LastTimeAddedMoney, NumberOfWins, gamesplayed...
Can do the same with nickname and moneyamount only... can maybe add "tableMoney" that is the total money he entered to the table with.